nicegui/nicegui.py

```diff
@@ -19,8 +21,9 @@
 from .helpers import safe_invoke
 from .page import page
 
-globals.app = app = App()
-globals.sio = sio = SocketManager(app=app)._sio
+globals.app = app = App(default_response_class=json.NiceGUIJSONResponse)
+socket_manager = SocketManager(app=app, json=json)  # custom json module (wraps orjson)
```

Comment:
For symmetry I would write:
```py
globals.sio = sio = SocketManager(app=app, json=json)._sio
```

Reply:
Here I disagree.  `SocketManager`'s `_sio` is a private/internal variable, so preferably we do not use it.
`SocketManager` adds the property `sio` to the passed `app` instance, so that's a public API, that's why I think it's better to go with `app.sio` instead.

```
class SocketManager:
    """
    Integrates SocketIO with FastAPI app.
    Adds `sio` property to FastAPI object (app).
...
```

Reply:
Good point!